xml/System.Net.PeerToPeer.Collaboration/PeerApplication.xml

```diff
@@ -415,7 +415,7 @@
       <Docs>
         <param name="other">The <see cref="T:System.Net.PeerToPeer.Collaboration.PeerApplication" /> instance to test for equality.</param>
         <summary>Compares the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerApplication" /> to the specified <see cref="T:System.Net.PeerToPeer.Collaboration.PeerApplication" /> instance.</summary>
-        <returns>Returns <see langword="true" /> if the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerApplication" /> instance provided has matching data, else <see langword="false" />. This method also returns <see langword="false" /> if <paramref name="other" /> is <see langword="null" />.</returns>
+        <returns><see langword="true" /> if the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerApplication" /> instance provided has matching data, else <see langword="false" />. This method also returns <see langword="false" /> if <paramref name="other" /> is <see langword="null" />.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerApplication" /> instance provided has matching data; otherwise, <see langword="false" />. This method also returns <see langword="false" /> if <paramref name="other" /> is <see langword="null" />.</returns>
```